arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts

```diff
@@ -304,6 +304,7 @@
 		touchscreen-size-y = <1280>;
 		avdd-supply = <&reg_tsp_3p3v>;
 		vdd-supply = <&reg_tsp_1p8v>;
+		linux,keycodes = <KEY_MENU KEY_BACK>;
```

Comment:
Based on your messages on Matrix I think you said it should be `KEY_APPSELECT` for this device, no?